migration-guide.md

```diff
@@ -1,74 +1,7 @@
-akka-kryo-serialization - migration guide
+pekko-kryo-serialization - migration guide
 =========================================
 
-Migration from 2.4.x to 2.5.x
+Migration from akka-kryo-serialization to pekko-kryo-serialization
 -----------------------------
-* `EnumerationSerializer` has been deprecated with 2.4.2, with 2.5.0 default serializer for `scala.Enumeration` has been switched to `EnumerationNameSerializer`, which is not backwards compatible.  
+* todo
```

Comment:
Should at least upgrade to latest version of akka-kryo-serialization before migrating to pekko-kryo-serialization